.github/ISSUE_TEMPLATE/01-building-an-application.yml

```diff
@@ -10,20 +10,21 @@ body:
   - type: dropdown
     id: android-type
     attributes:
-      label: Android application type
-      description: In what type(s) of Android application(s) do you see this issue?
+      label: Android framework version
+      description: In what target framework(s) of Android application(s) do you see this issue?
       multiple: true
       options:
-      - Classic Xamarin.Android (MonoAndroid13.0, etc.)
-      - .NET Android (net7.0-android, net8.0-android, etc.)
+      - net8.0-android
+      - net9.0-android
```

Comment:
Could these just say ".NET 8" and ".NET 9"?

Reply:
I went with this because the .NET version you install isn't necessarily the bits you are using.  That is, if you install .NET 9 and target `net8.0-android`, you are actually using the .NET 8 bits but customers don't know that.  I thought it might be more helpful for us to know what TF they are actually using.

That said, I don't have a strong opinion either way.  🤷